### PR TITLE
Copy directory structure recursive

### PR DESCRIPTION
**Pull request by aivils**

### CODE REVIEW

File: post-release-buildpack [hunk 1]
```diff
@@ -6,13 +6,18 @@ verify_app_name "$APP"
...
+    elif [[ -f $path ]]; then
+      dokku_log_verbose "- Copying $subpath"
+      id=$(docker run -i -a stdin $IMAGE /bin/bash -c "cat > /app/$subpath" < "$path")
+    fi
     test "$(docker wait $id)" -eq 0
     docker commit $id $IMAGE > /dev/null
-    docker stop $id > /dev/null 2>&1 || true
-    docker kill $id > /dev/null 2>&1 || true
+    # docker stop $id > /dev/null 2>&1 || true
+    # docker kill $id > /dev/null 2>&1 || true
```
Comment (josegonzalez) [inline post-release-buildpack:21]: Why this?

Comment (aivils) [reply post-release-buildpack:21]: It looks pretty weird. I am not a docker expert. I cannot found similar code in the similar plugins. Can You explain for what is stop and kill?

Comment (josegonzalez) [reply post-release-buildpack:21]: It stops/kills the container dangling container that we created for the copy step: https://docs.docker.com/engine/reference/commandline/stop/

File: post-release-dockerfile [hunk 2]
```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
```
Comment (josegonzalez) [inline post-release-dockerfile:1]: I would remove this completely. If you're using a dockerfile for deploys, you should be shipping all the code that you need for the docker image within your repository. I'm not merging the code with this change, full-stop.

**Review by josegonzalez — CHANGES_REQUESTED** [review]

File: post-release-dockerfile (hunk 2 above)
Comment (aivils) [reply post-release-dockerfile:1]: Okay, I don't need this plugin without Dockerfile.

**Comment by aivils** [comment]
Unfortunately I am not sure Dockerfile can access anything outside git repo provided by dokku. This way looks for me very powerful. It copies anything without weird rules. Yes my project have Dockerfile. Yes my project does not have private docker image.
As a result I cannot imagine simple way copy files from host's absolute path.

**Pull request closed without merging**